hub/core/chunk/base_chunk.py

```diff
@@ -357,6 +366,19 @@ def _pop_sample(self):
         self.shapes_encoder._pop()
         self.byte_positions_encoder._pop()
 
+    def pop_multiple(self, num_samples):
+        self.prepare_for_write()
+
+        self.data_bytes = self.data_bytes[
+            : self.byte_positions_encoder[
+                self.shapes_encoder.num_samples - num_samples
+            ][0]
+        ]
+
+        for _ in range(num_samples):
+            self.shapes_encoder._pop()
+            self.byte_positions_encoder._pop()
```

Comment:
have a try catch here around Index error, sometimes encoders won't be able to pop due to chunkcompression or none appends